packages/aws-rfdk/docs/upgrade/upgrading-0.38.md

```diff
@@ -0,0 +1,208 @@
+# Upgrading to RFDK v0.38.x or Newer
+
+Starting in RFDK v0.38.0, [Deadline Secrets Management](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html) will be enabled on the
+`Repository` and `RenderQueue` constructs by default when using Deadline 10.1.18 and later. This can cause some issues with existing farm configurations because Deadline Secrets Management requires:
+
+- The internal protocol on the Render Queue to be HTTPS
+- The external protocol on the Render Queue to be HTTPS
+
+Additionally, if you plan on upgrading to RFDK 0.38 or newer and Deadline 10.1.18 or newer, but do **not** want to enable Deadline Secrets Management, you may have to make changes to your RFDK application so that
```

Comment:
```suggestion
Additionally, if you plan on upgrading to RFDK 0.38 or newer and Deadline 10.1.19 or newer, but do **not** want to enable Deadline Secrets Management, you may have to make changes to your RFDK application so that
```

Reply:
Updated